quartz/plugins/transformers/opengraphImage.ts

```
@@ -0,0 +1,99 @@
+              } else {
+                // Priority 3: Use default image
+                opengraphImage = opts.defaultImage
+              }
+            }
+            
+            // Convert relative URLs to absolute paths from site root
+            if (opengraphImage && isRelativeURL(opengraphImage)) {
+              // If the image starts with "/" it's already relative to root, keep as-is
+              if (opengraphImage.startsWith("/")) {
+                // Already absolute path from root
+                opengraphImage = opengraphImage
```

Biome (2.1.2)

[error] opengraphImage is assigned to itself.

This is where is assigned.

Self assignments have no effect and can be removed.

(lint/correctness/noSelfAssign)